### ISSUE TITLE
Is it possible to load by a mouseover event a label from annotation?

### ISSUE THREAD

**Issue by daenbol** [dsehnal/LiteMol#67]
Is it possible in a Litemol to load by a mouseover event a label from annotation? Not just a custom one
![image](https://user-images.githubusercontent.com/53538336/110156728-61235900-7df8-11eb-9a30-9539a500eb09.png)
Something like here from Mol*
![image](https://user-images.githubusercontent.com/53538336/110157008-bbbcb500-7df8-11eb-8ed4-17929298d922.png)

**Comment by dsehnal** [comment]
Yes, Mol* builds a lot on what was done in LiteMol. In LiteMol it's done by registering additional ``https://github.com/dsehnal/LiteMol/blob/master/src/lib/Bootstrap/Interactivity/Highlight.ts``.

I would recommend you use Mol* instead of LiteMol tho.
